phongo_version.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2014-2018 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef PHONGO_VERSION_H
+#define PHONGO_VERSION_H
+
+#define PHP_MONGODB_VERSION "1.5.0-dev"
+#define PHP_MONGODB_STABILITY "devel"
+#define PHP_MONGODB_VERSION_DESC 1,5,0,0
```

Comment:
Instructions for bumping the version in `CONTRIBUTING.md` should be updated in the same commit (assuming that still refers to `php_phongo.h`).

Reply:
OK, will fix that (thought I had).